runtime/test/test_runtime.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import tempfile
+import unittest
+from pathlib import Path
+
+import torch
+from executorch.extension.pybindings.portable_lib import Verification
+
+from executorch.extension.pybindings.test.make_test import (
+    create_program,
+    ModuleAdd,
+    ModuleMulti,
+)
+from executorch.runtime import Runtime
+
+
+class RuntimeTest(unittest.TestCase):
+    def test_smoke(self):
+        ep, inputs = create_program(ModuleAdd())
+        runtime = Runtime.get()
+
+        program = runtime.load_program(
```

Comment:
Please add a test that get() really returns a singleton.

```suggestion
        runtime = Runtime.get()

        # Demonstrate that get() returns a singleton.
        runtime2 = Runtime.get()
        self.assertTrue(runtime is runtime2)

        program = runtime.load_program(
```